src/libaktualizr/storage/storage_common_test.cc

```diff
@@ -571,7 +571,7 @@ TEST(storage, list_remove_targets) {
   EXPECT_EQ(tfs.size(), 0);
   EXPECT_FALSE(boost::filesystem::exists(temp_dir.Path() / "images" / "HASH"));
 }
-
+#endif
```

Comment:
Can these tests get moved elsewhere and restored?

Reply:
I've added two tests to packagemanagerfake, which cover the same functionality.